src/health_misinfo_shared/fine_tuning.py

```diff
@@ -31,6 +32,58 @@
 UNCHECKWORTHY_EXPLANATIONS = ["nothing to check", "hedged claim"]
 VALID_EXPLANATIONS = CHECKWORTHY_EXPLANATIONS + UNCHECKWORTHY_EXPLANATIONS
 
+CATEGORY_WEIGHTS = {
+    "understandability": 15,
+    "type_of_claim": 10,
+    "type_of_medical_claim": 5,
+    "support": 7,
+    "harm": 10,
+}
+LABEL_SCORES = {
+    "understandability": {"understandable": 1, "vague": -3, "not understandable": -10},
+    "type_of_claim": {
+        "opinion": -2,
+        "personal": -3,
+        "citation": 5,
+        "hedged claim": -3,
+        "statement of fact": 3,
+        "advice/recommendation": 3,
+        "promotion": -10,
+        "not a claim": -10,
+    },
+    "type_of_medical_claim": {
+        "symptom": 5,
+        "cause/effect": 5,
+        "correlation": 3,
+        "prevention": 5,
+        "treatment/cure": 5,
+        "outcome": 3,
+        "statistics": 3,
+        "not medical": -10,
+    },
+    "support": {
+        "uncontroversial statement": -5,
+        "disputed claim": 7,
+        "widely discredited": 10,
+        "novel claim": 7,
+        "can't tell": 0,
+    },
+    "harm": {
+        "high harm": 15,
+        "some harm": 10,
+        "low harm": 5,
+        "indirect harm": 5,
+        "harmless": -10,
+        "can't tell": 0,
+    },
+}
+SUMMARY_THRESHOLDS = {
+    # MAKE SURE THESE ARE ALWAYS IN DESCENDING ORDER
```

Comment:
Might be worth explicitly sorting by value, either here or when reading the values?